test/e2e/tests/ngJqSpec.js

```diff
@@ -0,0 +1,12 @@
+describe('Customizing the jqlite / jquery version', function() {
+
+  it('should be able to force jqlite', function() {
+    loadFixture("ngJq").andWaitForAngular();
+    expect(browser.executeScript('return window.angular.element !== window.jQuery')).toBe(true);
+  });
```

Comment:
It might be a good idea to also test that `window.jQuery` is defined.

E.g. imagine 'jquery.js' isn't properly loaded in the future for whatever reason (e.g. directory is moved). I would be good to catch this in the test (and fix).